README.md

```diff
@@ -63,6 +63,25 @@ parseDefinition(componentsDefinition).then((componentSet) => {
 });
 ```
 
+### processInfo
```

Comment:
The example could be improved I guess. I think you already need quite deep knowledge of the code to understand what is expected as parameters and what the functions return as result. Maybe it's an idea to describe the input and output for both?

Reply:
I updated the name and the structure. I left the `processTemplates` and `processInfo` function out of the documentation as these probably aren't used by integrators.